tests/check/check_h26xsigned_sign.c

```diff
@@ -563,17 +562,24 @@ START_TEST(recurrence)
   nalu_list_check_str(list, "GIPPGIPPGIPPGIPPGIPPGIPPGI");
 
   nalu_list_item_t *item;
-  int gop_counter = 0;
-  for (int i = 1; i <= (list->num_items); i++) {
+  int last_sei_size = 0;
```

Comment:
You have made a mistake when resolving this section.

Reply:
Good catch. Also the description for this test needs to be changed. It's still about GOPs instead of frames.